ServiceBased/README.md

```diff
@@ -10,6 +10,15 @@ These sample tools are script tools that utilize the Python API to perform vario
    - The AdminTools.zip contains script tools that allow workflow admins to import, export, and transfer workflows.
    - The RecPost_Tool.zip contains a script tool that may be published as a geoprocessing service to reconcile and post branch versions.
 
+* [Jupyter Notebook Examples](./Jupyter%20Notebook%20Examples/) - These tools are compatible with ArcGIS Workflow Manager versions 10.9 and 10.9.1.
```

Comment:
Do we need to do anything to get them working for 11.0 (and 11.1)? Or do we just need to test them?